app/assets/javascripts/components/Columns/Columns.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Column from './ColumnItem';
+import PropTypes from 'prop-types';
+
+const Columns = ({
+  columns
+}) =>
+  columns.map(column =>
+    <Column
+      title={column.title}
+      renderAction={column.renderAction}
+      key={column.title}
+    >
+      { column.children }
+    </Column>
+  )
+
+Columns.propTypes = {
+  columns: PropTypes.array.isRequired
```

Comment:
Do we have a shape of a column to use it like this?

```js
columns: PropTypes.arrayOf(columnShape).isRequired
```

Reply:
Nope, but I will create :smile: